tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -396,3 +396,20 @@ def test_jumpstart_model_with_deployment_configs(setup):
     response = predictor.predict(payload, custom_attributes="accept_eula=true")
 
     assert response is not None
+
+
+def test_jumpstart_session_with_config_name():
+    model = JumpStartModel(model_id="meta-textgeneration-llama-2-7b")
+    assert model.config_name is not None
+    session = model.sagemaker_session
+
+    with mock.patch("botocore.client.BaseClient._make_request") as mock_make_request:
+        try:
```

Comment:
Just for my understanding, what is this `try`/`except` ?

Reply:
still, I'm not sure why you need a `try` / `except`, particularly since you are mocking this.

Consider either removing or adding an inline comment to explain that.